docs/src/explanation/buses_type_explanation.md

```diff
@@ -0,0 +1,46 @@
+# [Understanding ACBusTypes](@id bustypes)
+
+`PowerSystems.jl` supports multiple types of AC buses, [listed here](@ref acbustypes_list).
+When creating nodal datasets, the definitions for AC Buses can have a significant impact on the
+topology logic for the network.
+
+## Voltage Control Types
+
+  - `PQ`:
+    
+      + **Known:** Real Power Injection ($P$) and Reactive Power Injection ($Q$). These are typically the loads at that bus or fixed power factor generators.
+      + **Unknown:** Voltage Magnitude ($|V|$) and Voltage Angle ($\delta$).
+      + Represents a bus where the voltage magnitude and angle are free to vary based on the system conditions.
+
+  - `PV`:
+    
+      + **Known:** Real Power Injection ($P$) and Voltage Magnitude ($|V|$).
+      + **Unknown:** Reactive Power Injection ($Q$) and Voltage Angle ($\delta$).
+      + Typically represents a bus with an injector connected, where the injector controls the reactive power output and regulates the bus voltage magnitude to a setpoint.
+
+## Reference and Slack Buses
+
+There is a nuanced distinction between a slack bus and a reference bus. In most small test sytems and
+academic discussions, the system has a single slack bus which is also the reference bus. However, for large interconnected cases or cases with a very radial structure, having a single bus that takes on all the real power mistmatch in the system can lead to erroneous results.
```

Comment:
nitpick: would be nice to be more explicit about how something is classified if it's both a slack and a reference bus